.gitignore

```diff
@@ -40,4 +40,7 @@ pip-delete-this-directory.txt
 # VirtualEnv
 .venv/
 
+# Developers
 *.sw*
+manage.py
```

Comment:
@aidanlister must have a custom one.

We're not using it to start the app anyways (`$ django-admin.py runserver --settings=example.settings`)